pkg/info/server_info_test.go

```diff
@@ -52,9 +52,14 @@ func Test_WaitUntilReady(t *testing.T) {
 }
 
 func Test_Read_Write(t *testing.T) {
-	filepath := os.TempDir() + "/server-info"
+	filepath := os.TempDir() + "server-info"
 	defer os.Remove(filepath)
-	info := &ServerInfo{Protocol: UDS, Language: Go, Version: ""}
+	info := &ServerInfo{
+		Protocol: TCP,
+		Language: Java,
+		Version:  "11",
+		Metadata: map[string]string{"key1": "value1", "key2": "value2"},
+	}
```

Comment:
I update this line to match unit test in Java SDK such that I can verify the output files are identical between Go and Java. Feel convenient to keep it.